packages/instant/src/components/zero_ex_instant.tsx

```diff
@@ -79,6 +80,8 @@ export class ZeroExInstant extends React.Component<ZeroExInstantProps> {
         // tslint:disable-next-line:no-floating-promises
         asyncData.fetchAndDispatchToStore(this._store);
         // tslint:disable-next-line:no-floating-promises
+        gasPriceEstimator.getFastAmountInWeiAsync();
```

Comment:
Just some thoughts  -- there is the question of whether this information should be dispatched to the store. However, since we are not displaying this value I think this is a good approach.